coreos-kubernetes-multimaster/azuredeploy.json

```diff
@@ -0,0 +1,832 @@
+          "linuxConfiguration": {
+            "disablePasswordAuthentication": true,
+            "ssh": {
+              "publicKeys": [
+                {
+                  "path": "[variables('sshKeyPath')]",
+                  "keyData": "[parameters('sshKeyData')]"
+                }
+              ]
+            }
+          }
+        },
+        "storageProfile": {
+          "imageReference": {
+            "publisher": "CoreOS",
+            "offer": "CoreOS",
+            "sku": "[parameters('imageSku')]",
+            "version": "latest"
+          },
+          "osDisk": {
+            "name": "[concat(variables('workerVmNamePrefix'), copyindex())]",
+            "vhd": {
+              "uri": "[concat('http://',variables('workerStorageAccountName'),'.blob.core.windows.net/vhds/',variables('workerVmNamePrefix'), copyindex(),'.vhd')]"
```

Comment:
2 things:

1) "all clouds" uri best practice

2) For performance reasons, you should only put up to 20 VM disks in the same storage account (40 if you're edgy). Either limit the number of worker VMs or specify multiple storage accounts for the OS disks